spec.html

```diff
@@ -369,52 +374,61 @@ <h1>
 				1. Let _runningContext_ be the running execution context.
 				1. If _runningContext_ is not already suspended, suspend _runningContext_.
 				1. Push _evalContext_ onto the execution context stack; _evalContext_ is now the running execution context.
-				1. Perform HostImportModuleDynamically(*null*, _specifierString_, _innerCapability_).
+				1. Let _referrer_ be the Realm component of _evalContext_.
+				1. Perform HostLoadImportedModule(_referrer_, _specifierString_, ~empty~, _innerCapability_).
```

Comment:
re. this, https://github.com/nodejs/node/pull/48655#discussion_r1335771534 was brought to my attention.

Reply:
I can definitely separate this out into another PR, but I think we will need to change at least _something_ here; HostImportModuleDynamically allowed passing null, but HostLoadImportedModule does not. Let's discuss that.

Reply:
→ #390